sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/DateExpressionsSuite.scala

```diff
@@ -741,4 +742,16 @@ class DateExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     test("2015-07-24 00:00:00", null, null)
     test(null, null, null)
   }
+
+  test("SPARK-22716: UnixTimestamp should not use global variables") {
```

Comment:
We don't need a lot of this kind of tests, just one test to make sure `ctx.addReferenceObj` doesn't add global variable.